src/features/counter/counter.tsx

```diff
@@ -7,8 +7,8 @@ export const Counter = () => {
   const dispatch = useAppDispatch();
 
   return (
-    <div className="mx-auto flex max-w-sm flex-col items-center rounded-lg bg-jod-white p-6 shadow">
-      <span className="text-xl font-medium text-jod-black">{count}</span>
+    <div className="mx-auto flex max-w-sm flex-col items-center rounded-lg bg-[#ffffff] p-6 shadow">
+      <span className="text-xl font-medium text-[#000000]">{count}</span>
```

Comment:
Just nitpicking and as this component was just for testing purposes, but should it be now possible to directly use `bg-white`& `text-black` ?